CMakeLists.txt

```diff
@@ -331,15 +333,13 @@ else()
 	if (PTHREAD_LIBRARY)
 		message(STATUS "Pthread library: ${PTHREAD_LIBRARY}")
 	else()
-		if (NOT "${CMAKE_PREFIX_PATH}" STREQUAL "")
-			message(STATUS "WARNING: cross-compiler pthread not found - fallback to POSIX default")
-			set (PTHREAD_LIBRARY -pthread)
-			set (PTHREAD_INCLUDE_DIR "")
-		else()
-			message(FATAL_ERROR "Failed to find pthread library. Specify PTHREAD_LIBRARY.")
-		endif()
+		message(FATAL_ERROR "Failed to find pthread library. Specify PTHREAD_LIBRARY.")
```

Comment:
Please restore this code that relies on set CMAKE_PREFIX_PATH, it was here for a reason (crosscompiler on ARM), and your solution doesn't seem to use this path.

Reply:
I think your hack will break Android.. I pushed a slightly updated version that uses set(PTHREAD_LIBRARY ${CMAKE_THREAD_LIBS_INIT})

I hate cmake

Reply:
Welcome to the club - although all others are either much worse or at an early development stage. :).

Your fix seems to be ok, the build on ARM cross is ok, can even compile "testing" stuff.